src/translations/en.json

```diff
@@ -511,6 +512,9 @@
   "NoAccessPage.postListings.schemaTitle": "No publishing rights",
   "NoAccessPage.postListings.heading": "You can't publish listings",
   "NoAccessPage.postListings.content": "You need to receive publishing rights from the {marketplaceName} team.",
+  "NoAccessPage.initiateTransactions.schemaTitle": "No transaction rights",
+  "NoAccessPage.initiateTransactions.heading": "You don’t have transaction rights",
+  "NoAccessPage.initiateTransactions.content": "You need to receive transaction rights from the {marketplaceName} team.",
```

Comment:
I'm confused with the back-and-forth regarding the copy texts, but here I believe that it is not using Katri's final version at https://whimsical.com/permissions-access-control-LRn1pRNPJ65xhReX5pgcAs@2bsEvpTYSt1His9DMeWRBED1VDEdofji3fK which, at the moment of writing, is `To start a transaction, you need to receive transaction rights from the {marketplaceName} team.` (the first part of the sentence is missing).

Reply:
Or maybe this is something that will get updated in the CTA project, but I'm not certain of that, so maybe better to have the final text added now already?